### DISCUSSION TITLE
GPU and GPU0 in system information window

### DISCUSSION THREAD

**Discussion by MagicAndre1981** [winsiderss/systeminformer#1826] [Questions / Answers] [answered]
I just noticed that I have 2 GPU entries in system information window and in both the values are a different for some data (shared mem and GPU0 has extra data in shared graph):

![image](https://github.com/winsiderss/systeminformer/assets/8274816/066d263a-7fed-48bb-8a04-aef9699e32ae)

![image](https://github.com/winsiderss/systeminformer/assets/8274816/1fa4f52f-3c47-417c-8e4c-f0353a225862)

![image](https://github.com/winsiderss/systeminformer/assets/8274816/63414220-bc7e-4708-a446-bb165607f8e4)

Any ideas?

this is a low end AMD Ryzem 3 5300U laptop with only Vega iGPU and latest AMD chipset and GPU drivers under Win10 19045.3324 x64 which is enough here for typing on github

![image](https://github.com/winsiderss/systeminformer/assets/8274816/8096b4a3-f96d-4569-9160-25ac5442f052)

![image](https://github.com/winsiderss/systeminformer/assets/8274816/5d29e093-bf02-4300-967f-1389dbcd6b6a)

**Comment by dmex** [comment, marked as answer]
> I just noticed that I have 2 GPU entries in system information window

The first entry is always visible (enabled by default) and shows combined totals for all graphics cards. The second entry (disabled by default) is only visible when you select individual graphics cards from the Options > Graphics Devices window and the "0" from "GPU 0" is the PCI Bus Location of the graphics adapter.

> the values are a different for some data (shared mem and GPU0 has extra data in shared graph)

The Graphics Devices window (GPU0-5) is using the newer Windows 10/11 D3DKMT-V2 statistics while the default window (GPU) is using the legacy Windows Vista D3DKMT-V0 statistics. The default entry (GPU) includes all graphics cards (including the default software-only rendering adapter) and values will generally be different (especially when programs are rendering using the software adapter). 

The "Shared memory limit" shown on V2 provider (GPU0-5) graphs is "Commit limit" which is not the same as the "SharedSystemMemorySize" used as the "Shared memory limit" for the legacy graphs and statistics. The commit statistics are arguably more interesting and ideally will replace the "shared" values (if they're reliable) hence why there's multiple commit statistics in the "shared memory" graph and commit totals instead of shared totals.

**Reply by MagicAndre1981** [reply]
thanks for the explanation, add some information to the option page to explain the option, there is a lot of space.
